oca_dependencies.txt

```diff
@@ -0,0 +1,3 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+account-operating-unit https://github.com/eficent/operating-unit 12.0-mig-account_operating_unit
```

Comment:
Hi @bjeficent account_operating_unit is merged. We have to remove this line.

Then it will be ready to merge ad comments from @rvalyi were attended.

Reply:
Actually account_operating_unit is not merged :sweat_smile:  let's wait a bit